languages/nl/Zend_Captcha.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright  Copyright (c) 2005-2016 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license    http://framework.zend.com/license/new-bsd     New BSD License
+ */
+
+/**
+ * EN-Revision: 30.Jul.2011
+ */
+return [
+    // Zend\Captcha\ReCaptcha
+    "Missing captcha fields" => "Missende captcha velden",
+    "Failed to validate captcha" => "Captcha validatie mislukt",
+    "Captcha value is wrong: %value%" => "Captcha waarde is incorrect: %value%",
+
+    // Zend\Captcha\Word
+    "Empty captcha value" => "Lege captcha waarde",
+    "Captcha ID field is missing" => "Captcha ID veld mist",
```

Comment:
Nevermind, I commented it earlier but send the review a bit later, see @derickr's discussion. I think the latter wins?